source/Cosmos.HAL2/Global.cs

```diff
@@ -46,14 +47,35 @@ static public void Init(TextScreenBase textScreen, bool InitScrollWheel, bool In
             // system level and not accessible from Core. Need to think about this
             // for the future.
             Console.Clear();
-            Console.WriteLine("Finding PCI Devices");
-            debugger.Send("PCI Devices");
-            PCI.Setup();
 
             Console.WriteLine("Starting ACPI");
             debugger.Send("ACPI Init");
             ACPI.Start();
 
+            Console.WriteLine("Finding PCI Devices");
```

Comment:
I don't think we should write as much to the console, since the user cant control this behavior.